packages/expo-branch/unimodule.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "expo-branch",
+  "platforms": ["android"],
+  "android": {
+    "packages": ["expo.modules.branch.BranchPackage"]
```

Comment:
it's not required to specify package here
https://github.com/unimodules/react-native-unimodules/blob/master/gradle.groovy#L61

Reply:
Apart from `BranchPackage < org.unimodules.Package` we also have `vendored.RNBranchPackage < com.facebook.react.Package` in the same module, which introduces a conflict. Both removing the first one or modifying the second one would require manual changes when updating the vendored code.

https://github.com/unimodules/react-native-unimodules/blob/09234cff13d179c74cf347136c04a477621c9fc7/gradle.groovy#L66-L68